### ISSUE TITLE
Alignment looks off in the pre-publish panel

### ISSUE THREAD

**Issue by mtias** [WordPress/gutenberg#18249]
![image](https://user-images.githubusercontent.com/548849/68075663-06fd2880-fdab-11e9-921f-8a6cc3955468.png)

**Comment by paaljoachim** [comment]
I just tested locally using Desktop Server, Chrome and Gutenberg version 7.5
This is what I see:
![Screen Shot 2020-02-18 at 18 44 24](https://user-images.githubusercontent.com/5323259/74762789-d91af180-527e-11ea-8f43-6e932628e5e6.png)

![Screen Shot 2020-02-18 at 18 44 31](https://user-images.githubusercontent.com/5323259/74762805-dd470f00-527e-11ea-814b-a329d9a277dd.png)

I am thinking this issue has been fixed in one of the Guteneberg versions after the initial issue was created.

**Comment by folletto** [comment]
I'd note it still feels oddly placed even when it's centered, as the × makes it still looks off.

I wonder if we should explore a bit more how the pre- and post-publish panels work and are aligned. 

Maybe something like replacing the × in both pre/post with two buttons that take the full top area:

`| [   Cancel   ] [   Publish   ]|` vs  `|          [ Publish ]     × |`

**Comment by mapk** [comment]
A few of us in today's Design triage in slack tested this out again. We remembered that this was recently solved with this PR: https://github.com/WordPress/gutenberg/pull/17726

After testing again, I did not notice any alignment issues.

![Screen Shot 2020-02-18 at 9 45 18 AM](https://user-images.githubusercontent.com/617986/74762984-bb746a80-5233-11ea-9c21-f51ba5804c04.png)

There was some desire to iterate more on the placement which can be done as mentioned by @folletto above. I remember another issue/PR exploring the positioning further as well https://github.com/WordPress/gutenberg/issues/19175.

**Comment by mtias** [comment]
I'll close.

**Comment by paaljoachim** [comment]
Do we need a new issue for what Erin @folletto mentioned above?

> I wonder if we should explore a bit more how the pre- and post-publish panels work and are aligned.

Maybe something like replacing the × in both pre/post with two buttons that take the full top area:

| [ Cancel ] [ Publish ]| vs | [ Publish ] × |

**Comment by paaljoachim** [comment]
Followup:

Here is from a Norwegian site. (Yesterday I only checked a local English language site.)

<img width="441" alt="Screen Shot 2020-02-19 at 05 30 49" src="https://user-images.githubusercontent.com/5323259/74802076-04cac580-52d9-11ea-9a6b-33786fd4f1cd.png">

<img width="419" alt="Screen Shot 2020-02-19 at 05 31 11" src="https://user-images.githubusercontent.com/5323259/74802097-1ad88600-52d9-11ea-8409-c3363886e1ed.png">

Bottom line: The problem remains. There is actually a bigger fix that is needed. Which perhaps can be done in https://github.com/WordPress/gutenberg/issues/19175

**Comment by paaljoachim** [comment]
The problems remains but now it shows a little different.
<img width="378" alt="Screen Shot 2020-03-13 at 10 27 13" src="https://user-images.githubusercontent.com/5323259/76608022-3b30e480-6515-11ea-82c5-5174d2f28fcc.png">

We might as well reopen this issue. 
@mtias @mapk
